### ISSUE TITLE
iPad - Can’t delete (or perform other options) on action

### ISSUE THREAD

**Issue by bertke** [FlutterFlow/flutterflow-issues#1735]
### Has your issue been reported?

- [X] I have searched the existing issues and confirm it has not been reported.
- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

When on the iPad and editing an action, tapping on the three dots in the upper right corner of the action block doesn’t open the menu but rather just shows the help text.  

### Expected Behavior

In previous versions tapping those 3 dots brought up a dialog of options (delete, copy etc).  But now it just shows the help text that is normally shown on hover-over (on the desktop version) which is ‘Action Options’.

### Steps to Reproduce

1. Add an action to any widget
2. Go into the actions editor
3. Tap on the ellipses (the vertical 3 dots in the upper right corner)

This produces a small dialog that just says ‘Action Options’ rather than the options to delete/edit

### Reproducible from Blank

- [X] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

Can’t product on iPad (and can’t generate on desktop as I’m stopped by another bug reported by others)

### Context

I have an action I need to delete. It’s causing a bug which won’t let me continue testing/running the app

### Visual documentation

![IMG_0009](https://github.com/FlutterFlow/flutterflow-issues/assets/803569/3a55d0ae-a7a5-471a-ac70-0f83d3acdd9b)

### Environment

```markdown
- FlutterFlow version: latest iPad version
- Platform: iPad 
- Browser name and version:chrome
- Operating system and version affected: latest iOS (iPad)
```

**Comment by ToddHerron** [comment]
I’m having this issue as well.

**Comment by sbis04** [comment]
This should be fixed in the version `4.1.5` of the iPad app.

**Comment by ableinc** [comment]
Why is this closed when the issue is still happening on 4.1.6??
